app/models/booking.rb

```diff
@@ -11,5 +11,14 @@ class Booking < ActiveRecord::Base
   scope :active, -> { where(release_date_time: nil) }
   scope :inactive, -> { where.not(release_date_time: nil) }
   scope :last_week, -> { where('booking_date_time > ?', 1.week.ago) }
+  scope :bondable, -> { joins(:charges).merge(Charge.bondable) }
+
+  def bond_total
+    charges.map { |c| c.bond_amount || 0 }.reduce(:+)
```

Comment:
Can we outsource this math to the database? Might speed up the page, esp. if it's calculated for each booking.
http://apidock.com/rails/ActiveRecord/Calculations/ClassMethods/sum